package.json

```diff
@@ -97,6 +97,7 @@
     "babel-plugin-transform-react-remove-prop-types": "^0.4.21",
     "babel-plugin-unwrap-createStyles": "link:packages/babel-plugin-unwrap-createStyles",
     "chai": "^4.1.2",
+    "classnames": "^2.2.6",
```

Comment:
Was transitive dependency of:
- recharts
- react-draggable
- react-select
- react-virtualized

Reply:
It's a peer dependency of these packages? Why do we need to install it?

Reply:
No it's a peer dependency of `notisstack`. `notisstack` would fail if we would remove the packages I mentioned.

Reply:
And it is already installed. We're not adding any new packages to the bundle in this PR

Reply:
I think that notisstack should make it a dependency cc @iamhosseindhv.

Reply:
What about waiting for https://github.com/iamhosseindhv/notistack/issues/72 resolution?

Reply:
We can always remove it once it's resolved. After all it's just an entry in the package.json.

Reply:
`classnames` and `prop-types` should be dependencies of `notistack`. This will be published in the next version.